pkg/tcp_metrics/tcp/tcpinfo.go

```diff
@@ -130,4 +130,10 @@ type LinuxTCPInfo struct {
 	RcvOooPack uint32
 
 	SndWnd uint32
+	RcvWnd uint32
+
+	Rehash uint32
+
+	TotalRto           uint16
+	TotalRtoRecoveries uint32
```

Comment:
One quick question - from the latest `tcp_info` struct, there is also a `tcpi_total_rto_time ` field, curious why that isn't included?